de/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="de">
+<head>
+  <title>Frag nicht zum Fragen, frag einfach</title>
+  <meta property="og:title" content="Frag nicht zum Fragen, frag einfach" />
+  <meta property="og:type" content="website" />
+  <meta property="og:url" content="https://dontasktoask.com" />
+  <meta property="og:locale" content="de_DE" />
+  <link rel="stylesheet" type="text/css" href="/style.css">
+  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+</head>
+<body>
+  <main>
+    <h1>Frag nicht zum Fragen, frag einfach</h1>
+    <p>
+      Seitdem ich im Internet unterwegs bin, gibt es immer irgendwelche
+      Leute, die sowas schreiben:
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+        Irgendein Java-Experte hier?
+      </p>
+    </blockquote>
+    <p>
+      Das ist aus mehreren Gründen eine schlechte Form. Was die Person 

```

Comment:
```suggestion
      Das ist aus mehreren Gründen unpassend. Was die Person 
```

"bad form" does not translate literally in my opinion. "unpassend" or "unangebracht" probably fits better.